### ISSUE TITLE
Scroll requests with `from` should throw an exception

### ISSUE THREAD

**Issue by martijnvg** [elastic/elasticsearch#9373]
During merging of the shard top doc results the from is ignored, while it can be taken into account during the first scroll search request.

From the user list: https://groups.google.com/forum/?utm_source=footer#!msg/elasticsearch/FS1qGg_-JK8/byKMykzwFNkJ

**Comment by clintongormley** [comment]
`from` on a scroll request would be subject to the same limitations as for a normal search request, ie a large from value could use excessive amounts of memory.

**Comment by nik9000** [comment]
I noticed this while working on #15125! I was actually surprised it didn't work but then I thought about it and it makes sense. I think it'd be better if `from` on a scroll throw an error back to the user. Best would be to get it working but its pretty weird to think about.

**Comment by clintongormley** [comment]
Yeah, I'm unsure of the use case here.  When would you want to use `from` with a scroll?  Seems odd.

**Comment by martijnvg** [comment]
+1 for throwing an error if `from` is used in a scroll search. I don't know of a use case for this. The only reason this issue was reported, is that this used to work in a pre 1.2 release.

**Comment by colings86** [comment]
This looks like it was fixed in https://github.com/elastic/elasticsearch/pull/23015
